src/docgen/render/markdown-render.ts

```diff
@@ -337,17 +337,30 @@ export class MarkdownRenderer {
       header: { title: enu.displayName },
     });
 
-    md.table(this.createTable(enu.members));
-    md.split();
-
     if (enu.docs) {
-      md.docs(enu.docs);
+      md.docs(enu.docs, this.language);
     }
 
-    for (const em of enu.members) {
-      md.section(this.visitEnumMember(em));
+    md.section(this.visitEnumMembers(enu.members));
+
+    return md;
+  }
+
+  public visitEnumMembers(
+    enus: EnumMemberSchema[],
+  ): MarkdownDocument {
+    if (enus.length === 0) {
+      return MarkdownDocument.EMPTY;
     }
 
+    const md = new MarkdownDocument({ header: { title: 'Enum Members' } });
```

Comment:
"Enum" is redundant at this point.

```suggestion
    const md = new MarkdownDocument({ header: { title: 'Members' } });
```